Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Food/Containers/lunchbox.yml

```diff
@@ -0,0 +1,313 @@
+- type: entity
+  id: LunchboxGeneric
+  parent: [ BoxCardboard, BaseBagOpenClose ]
+  name: lunchbox
+  description: For on-the-go meal carrying needs.
+  components:
+  - type: Sprite
+    sprite: DeltaV/Objects/Storage/lunchbox.rsi
+    layers:
+    - state: generic
+    - state: generic-open
+      map: ["openLayer"]
+  - type: Item
+    size: Normal
+    shape: null
+    sprite: DeltaV/Objects/Storage/lunchbox.rsi
+    heldPrefix: generic
+  - type: Storage
+    maxItemSize: Normal
+    grid:
+    - 0,0,1,1
+    - 3,0,1,1
+    - 4,0,4,2
+  - type: PhysicalComposition
+    materialComposition:
+      Plastic: 100
+  - type: MeleeWeapon
+    damage:
+      types:
+        Blunt: 2
+    soundHit:
+      path: "/Audio/Weapons/click.ogg"
+  - type: StaticPrice
+    price: 10
+
+- type: entity
+  parent: LunchboxGeneric
+  id: LunchboxGenericFilledRandom
+  suffix: Filled, Random
+  name: pre-packed lunchbox
+  description: Packed with love.
+  components:
+  - type: StorageFill
+    contents:
+    - id: FoodPizzaArnoldSlice
+      orGroup: HealthyOrUnhealthyMain
+      prob: 0.5
+      amount: 2
+    - id: FoodApple
+      orGroup: HealthyOrUnhealthyMain
+      prob: 0.5
+    - id: DrinkWaterBottleFull
+      orGroup: HealthyOrUnhealthyDrink
+      prob: 0.5
+    - id: DrinkCartonOrange
+      orGroup: HealthyOrUnhealthyDrink
+      prob: 0.5
+    - id: FoodSnackCheesie
+      orGroup: HealthyOrUnhealthySnack
+      prob: 0.25
+    - id: FoodSnackChocolate
+      orGroup: HealthyOrUnhealthySnack
+      prob: 0.25
+    - id: FoodCarrot
+      orGroup: HealthyOrUnhealthySnack
+      prob: 0.5
+    - id: PaperWrittenNoteFromMumGeneric
+      prob: 0.1
+
+- type: entity
+  parent: LunchboxGeneric
+  id: LunchboxGenericFilledHealthy
+  suffix: Filled, Healthy
+  name: pre-packed lunchbox
+  description: Packed with healthy love.
+  components:
+  - type: StorageFill
+    contents:
+    - id: FoodApple
+    - id: DrinkWaterBottleFull
+    - id: FoodCarrot
+    - id: PaperWrittenNoteFromMumHealthy
+      prob: 0.1
+
+- type: entity
+  parent: LunchboxGeneric
+  id: LunchboxGenericFilledUnhealthy
+  suffix: Filled, Unhealthy
+  name: pre-packed lunchbox
+  description: Packed with love.
+  components:
+  - type: StorageFill
+    contents:
+    - id: FoodPizzaArnoldSlice
+      amount: 2
+    - id: DrinkCartonOrange
+    - id: FoodSnackCheesie
+      orGroup: UnhealthySnack
+      prob: 0.25
+    - id: FoodSnackChocolate
+      orGroup: UnhealthySnack
+      prob: 0.25
+    - id: PaperWrittenNoteFromMumUnhealthy
+      prob: 0.1
+
+- type: entity
+  parent: LunchboxGeneric
+  id: LunchboxCommand
+  suffix: Command
+  components:
+  - type: Sprite
+    layers:
+    - state: command
+    - state: command-open
+      map: [ "openLayer" ]
```

Comment:
i ended up with similar statenames & .rsi locally however i could not get it to work at all in one .rsi file, no idea how you did it but it's mystical lol